__tests__/app.test.js

```diff
@@ -0,0 +1,80 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require('../db/data/test-data/index');
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('/api/topics', () => {
+  describe('GET', () => {
+    it('responds with an array of topic objects, each with slug and description properties', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.topics).toEqual(topicData);
+        });
+    });
+  });
+});
+
+describe('/api/articles', () => {
+  describe('GET', () => {
+    it(`responds with an array of article objects, each with author, title, article_id, topic, created_at, votes, article_img_url and comment_count properties`, () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(12);
```

Comment:
It's great that you remembered to include a length assertion outside a forEach which contains expect statements 👍